packages/account/src/Types/common.type.ts

```diff
@@ -163,6 +151,39 @@ export type TServerError = {
     fields?: string[];
 };
 
+export type TPaymentMethodIdentifier = typeof IDENTIFIER_TYPES[keyof typeof IDENTIFIER_TYPES];
+
+export type TPaymentMethodInfo = {
+    documents_required: number;
```

Comment:
what does documents_required mean?

Reply:
Its an API prop that returns the number of documents that is to be uploaded for a particular PM